ee/modules/110-istio/monitoring/prometheus-rules/multicluster.yaml

```
@@ -52,3 +52,26 @@
           TOKEN="$(deckhouse-controller helper gen-jwt --private-key-path <(echo "$KEY") --claim iss=d8-istio --claim sub=$LOCAL_CLUSTER_UUID --claim aud=$REMOTE_CLUSTER_UUID --claim scope=private-multicluster --ttl 1h)"
           curl -H "Authorization: Bearer $TOKEN" {{$labels.endpoint}}
           ```
+
+
+    - alert: D8IstioRemoteClusterNotSynced
+      expr: istio_remote_cluster_up == 0
+      for: 5m
+      labels:
+        severity_level: "4"
+        tier: cluster
+      annotations:
+        plk_markup_format: markdown
+        plk_protocol_version: "1"
+        plk_create_group_if_not_exists__d8_istio_remote_clusters_sync: D8IstioRemoteClustersSync,tier=~tier,prometheus=deckhouse,kubernetes=~kubernetes
+        plk_grouped_by__istio_remote__d8_clusters_sync: D8IstioRemoteClustersSync,tier=~tier,prometheus=deckhouse,kubernetes=~kubernetes
+        summary: Istio remote cluster `{{$labels.cluster_id}}` is not synced.
+        description: |
+          The Istio control plane in cluster `{{$labels.istiod}}` cannot synchronize with remote cluster `{{$labels.cluster_id}}`.
```

```suggestion
          The Istio control plane instance `{{$labels.istiod}}` cannot synchronize with remote cluster `{{$labels.cluster_id}}`.
```
Did i get it right that iabels.istiod contains istiod pod name?